pkg/deploy/api/v1/defaults.go

```diff
@@ -12,6 +12,13 @@ func init() {
 	}
 
 	err := api.Scheme.AddDefaultingFuncs(
+		func(obj *DeploymentConfigSpec) {
```

Comment:
Because we should, going to put it in its own commit.

On Aug 21, 2015, at 8:30 AM, David Eads <notifications@github.com> wrote:

In pkg/deploy/api/v1/defaults.go
<https://github.com/openshift/origin/pull/4299#discussion_r37629330>:

> @@ -12,6 +12,13 @@ func init() {
>  	}
>
>  	err := api.Scheme.AddDefaultingFuncs(
> +		func(obj *DeploymentConfigSpec) {

I'm pretty sure this broke serialization tests. Why are defaulting this?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4299/files#r37629330>.